websites/#/라프텔/presence.ts

```diff
@@ -6,7 +6,6 @@ let prevData = "",
 	animeData: detail = {},
 	animeDataEpisode: episode = {};
 
-/* eslint-disable camelcase */
```

Comment:
@DarkVillager how did you fix this again?

Reply:
oh I thought you did, was it @theusaf something with an empty function or so so it wouldnt get deleted?

Reply:
https://github.com/PreMiD/Presences/blob/fcc689d2709c70084758151c7ee00c7822c6ac5c/websites/B/Board%20Game%20Arena/util.ts#L1